src/__swaps__/screens/Swap/Swap.tsx

```diff
@@ -103,9 +105,11 @@ const useMountSignal = () => {
 };
 
 const useCleanupOnUnmount = () => {
+  const { accountAddress } = useAccountSettings();
```

Comment:
can we change the type of `accountAddress` to `Address` in the return of this hook instead of casting?

Reply:
Probably not feasibly unless we want to make a ton of changes to a lot of places in the app

Reply:
😔

Reply:
What we could do is instead of casting here (and other instances like this going forward), we could utilize `@ts-expect-error` to suppress it and once we end up migrating the hook over to return `Address` type, ts would pick up on all the locations we've added these expect tags and require us to cleanup (basically just remove the `@ts-expect-error` comment). 

I'm a fan of that over casting fwiw, but this isn't related to this PR whatsoever so yeah.

Reply:
is `Address` not a subtype of `string` though?

Reply:
nvm ur right it's complaining

Reply:
If we're talking about the same `Address` type it's a template literal type using `string` generic. It resolves to `0x${string}`.

Reply:
easiest fix might be to allow `Address | string` within the `userAssets` store for now while we work on migrating fully over to `Address` types